src/main/resources/application.conf

```diff
@@ -76,6 +76,10 @@ backend {
 google {
   // Used by cromwell to make calls to JES and to perform GCS reads/writes.
   // The latter can be overriden if the "User Mode" is used (see below).
+  // Use "application-default" to use the default service account credentials.
+  // This is useful if you are running on a GCE VM and if you don't need
+  // user level access. See https://developers.google.com/identity/protocols/application-default-credentials
+  // for more info.
```

Comment:
Sure. I can add it, but it is option 1 in that link:

   1. The environment variable GOOGLE_APPLICATION_CREDENTIALS is checked.
   If this variable is specified it should point to a file that defines the
   credentials. The simplest way to get a credential for this purpose is to
   create a service account using the Google Developers Console
   <https://console.developers.google.com/> in the section *APIs & Auth*,
   in the sub-section *Credentials*. Create a service account or choose an
   existing one and select *Generate new JSON key*. Set the environment
   variable to the path of the JSON file downloaded.

Want me to add that to the comment?

On Wed, Dec 16, 2015 at 10:53 AM, Scott Frazer <notifications@github.com>
wrote:

> In src/main/resources/application.conf
> <https://github.com/broadinstitute/cromwell/pull/329#discussion_r47792879>
> :
>
> > @@ -76,6 +76,10 @@ backend {
> >  google {
> >    // Used by cromwell to make calls to JES and to perform GCS reads/writes.
> >    // The latter can be overriden if the "User Mode" is used (see below).
> > +  // Use "application-default" to use the default service account credentials.
> > +  // This is useful if you are running on a GCE VM and if you don't need
> > +  // user level access. See https://developers.google.com/identity/protocols/application-default-credentials
> > +  // for more info.
>
> You mentioned that there's something we have to do to use this on our
> desktops... I don't believe that's documented in the link you provided, is
> that worth adding here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/broadinstitute/cromwell/pull/329/files#r47792879>.
>